### PR TITLE
[Merged by Bors] - chore(Logic/ExistsUnique): amend usages of existsUnique to follow naming convention

### PR DESCRIPTION
**Pull request by b-mehta**
As a definition `ExistsUnique`, the spelling of usages should be `existsUnique` rather than `exists_unique`. The majority of applications already conform to this (96), but there are some (30) exceptions. This PR fixes those.

The large import is the import of Tactic.Alias to Logic.ExistsUnique, and is necessary if deprecations are to be performed here.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Dynamics/Newton.lean [hunk 1]
```diff
@@ -103,11 +103,11 @@ unit) then we may write `x` as a sum `x = n + r` where `n` is nilpotent and `r`
 Moreover, `n` and `r` are unique.
 
 This can be used to prove the Jordan-Chevalley decomposition of linear endomorphims. -/
-theorem exists_unique_nilpotent_sub_and_aeval_eq_zero
+theorem existsUnique_nilpotent_sub_and_aeval_eq_zero
```
Comment (j-loreaux) [inline Mathlib/Dynamics/Newton.lean:106]: This one is missing a deprecation.

**Review by j-loreaux — APPROVED** [review]
bors d+

File: Mathlib/Dynamics/Newton.lean (hunk 1 above)
Comment (b-mehta) [reply Mathlib/Dynamics/Newton.lean:106]: Aah you're right, thanks so much for catching this!

**Comment by b-mehta** [comment]
bors merge

**Pull request closed without merging**